index.js

```diff
@@ -145,14 +145,26 @@ class ClinicDoctor extends events.EventEmitter {
     const logoPath = path.join(__dirname, 'visualizer', 'app-logo.svg')
     const nearFormLogoPath = path.join(__dirname, 'visualizer', 'nearform-logo.svg')
     const clinicFaviconPath = path.join(__dirname, 'visualizer', 'clinic-favicon.png.b64')
+    const systemInfoPath = path.join(__dirname, 'visualizer', 'info.json')
 
     // Load data
     const paths = getLoggingPaths({ path: dataDirname })
 
+    const chunks = []
+    const systemInfoStream = fs.createReadStream(paths['/systeminfo'])
+      .on('data', (data) => {
+        chunks.push(data)
+      })
+      .on('end', () => {
+        fs.writeFile(systemInfoPath, chunks.toString(), 'utf-8', function () {
+        })
+      })
```

Comment:
Why this instead of just copying the file? The other (and more efficient way of doing this if it's really needed is to just pipe from one stream to another...) 

```js
const in = fs.createReadStream(paths['/systeminfo'])
const out = fs.createWriteStream(systemInfoPath)
stream.pipeline(in, out, (err) => {
  if (err) /** handle the error **/
})
```

Reply:
Really tho, I'm just not sure why this copy is necessary at all

Reply:
Yeah, I see that, it's not a very good solution, it was originally a problem with accessing the data file. But I've updated it there.